operations/bills.md

```diff
@@ -202,6 +255,39 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 | `Iban` | string  | optional | Enterprise IBAN (International Bank Account Number). |
 | `Bic` | string  | optional | Enterprise BIC (Bank Identifier Code). |
 
+### Bill address
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Line1` | string  | optional | First line of the address. |
+| `Line2` | string  | optional | Second line of the address. |
+| `City` | string  | optional | City of the address. |
+| `Postal Code` | string  | optional | Postal code of the address. |
+| `SubdivisionCode` | string  | optional | ISO 3166-2 code of the administrative division. |
+| `CountryCode` | string  | optional | ISO 3166-1 code of the country. |
+
+#### Legal Identifiers
+
+A key value pair where key determines type of legal identifier and value is the string value of that identifier
+
+* `TaxIdentifier`
+* `CompanyName`
+* `Siret`
+* `Siren`
+* `NafCode`
+* `RcsCode`
+* `LegalStatus`
+* `RegisteredCapital`
+* `CityOfRegistration`
+* `TradesDirectoryRegistrationNumber`
+* `ItDestinationCode`
+* `ItFiscalCode`
+* `ItLotteryCode`
+* `HungarianVatCode`
+* `HungarianCompanyName`
+* `HungarianTaxPayerIdentifier`
+
```

Comment:
Is this enum finite or not? If it can change in the future, it should be presented here like this.
```suggestion
* `HungarianCompanyName`
* `HungarianTaxPayerIdentifier`
* ...
```